typespec-tests/src/main/java/com/_specs_/azure/clientgenerator/core/usage/implementation/ModelInOperationsImpl.java

```diff
@@ -65,7 +65,7 @@ public interface ModelInOperationsService {
         @UnexpectedResponseExceptionType(value = ResourceNotFoundException.class, code = { 404 })
         @UnexpectedResponseExceptionType(value = ResourceModifiedException.class, code = { 409 })
         @UnexpectedResponseExceptionType(HttpResponseException.class)
-        Mono<Response<Void>> inputToInputOutput(@HeaderParam("accept") String accept,
+        Mono<Response<Void>> inputToInputOutput(@HeaderParam("Content-Type") String contentType,
```

Comment:
This is because I removed the [logic](https://github.com/Azure/autorest.java/blob/main/typespec-extension/src/code-model-builder.ts#L1141) to add accept header.  And content-type header is added by TCGC. 
@weidongxu-microsoft I'm not quite sure on why do we add accept header to this case, the response is `void`, do you think we need to add accept header in this case?

Reply:
Yeah, we don't need the `accept`, as the response is 204.